onnxruntime/core/providers/dnnl/subgraph/dnnl_pow.cc

```diff
@@ -45,8 +45,12 @@ void DnnlPow::CreatePrimitive(DnnlSubgraphPrimitive& sp, DnnlNode& node) {
   }
 
   // DNNL eltwise_pow is defined as alpha*x^beta. We don't use alpha so it is hard coded to 1.0
-  dnnl::eltwise_forward::desc elementwise_desc(dnnl::prop_kind::forward_inference, dnnl::algorithm::eltwise_pow, src_md, 1.0, beta);
-  dnnl::eltwise_forward::primitive_desc elementwise_pd(elementwise_desc, dnnl_engine);
+  dnnl::eltwise_forward::primitive_desc elementwise_pd(dnnl_engine, dnnl::prop_kind::forward_inference,
+                                                      dnnl::algorithm::eltwise_pow, src_md,
+                                                      dnnl::memory::desc(src_md.get_dims(),
+                                                                         src_md.get_data_type(),
+                                                                         dnnl::memory::format_tag::any),
```

Comment:
Could we make the memory desc on a separate line. I think it would improve readability.

i.e.

    auto src2_md = dnnl::memory::desc(src_md.get_dims(),
                                                               src_md.get_data_type(),
                                                                dnnl::memory::format_tag::any);
    dnnl::eltwise_forward::primitive_desc elementwise_pd(dnnl_engine, dnnl::prop_kind::forward_inference,
                                                                                           dnnl::algorithm::eltwise_pow, src_md,
                                                                                           1.0, beta);